.github/dependabot.yml

```diff
@@ -6,6 +6,8 @@ updates:
     schedule:
       interval: "daily"
     labels:
-      - dependencies
-      - go
+      - automation
+    reviewers:
+      - "mtojek"
+      - "jsoriano"
```

Comment:
I'd prefer to put a [github] team here so we can more easily manage notifications and we don't need need to change this if the people related to the project changes. But we can go on like this by now.

Reply:
Sure thing.

BTW, we are running a PoC with a `oncall` github team, so the person oncall need to subscribe to the team and remove other team members. The idea is to reduce the overhead/notifications for other team members when they are not on-call, but we are still at the very beginning of this evaluation :)

Reply:
Oh, this sounds great to ping with the person of call for issues related to builds/CI. 

> the person oncall need to subscribe to the team and remove other team members.

Would it be possible to do it automatically by checking the Pagerduty calendars?

Reply:
I just answered you in the initial discussion we have opened in the obs-robots team regarding this process :)

Reply:
Thanks!

Reply:
```suggestion
```